packages/mongo/tests/observe_changes_tests.js

```
@@ -519,8 +519,8 @@ if (Meteor.isServer) {
       const [resolver1, promise1] = getPromiseAndResolver();
       const [resolver2, promise2] = getPromiseAndResolver();
 
-      await self.insert({x: 2, y: 3});
       self.expects.push(resolver1, resolver2);
+      await self.insert({x: 2, y: 3});
```

ESLint

[error] Replace `x:·2,·y:·3` with `·x:·2,·y:·3·`

(prettier/prettier)